gulpfile.js

```diff
@@ -11,12 +11,11 @@ var gulp = require('gulp'),
     concat = require('gulp-concat'),
     cache = require('gulp-cache'),
     babel = require("gulp-babel"),
-    gutil = require('gulp-util'),
     sass = require('gulp-sass'),
     cssmin = require('gulp-cssmin'),
     del = require('del');
 
-
+/*    gutil = require('gulp-util'), */
 
```

Comment:
i had some performance impacts and was trying to isolate the problem..  
anywayl look like i pushed a lot of crap on github last time. sry guys!